gtg.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exec ./scripts/debug.sh
```

Comment:
Can you put there a comment why this file was created? (hint: make it more discoverable for new contributors)